src/FINDOLOGIC/Api/RequestBuilders/Xml20/NavigationRequestBuilder.php

```diff
@@ -1,11 +1,10 @@
 <?php
 
-namespace FINDOLOGIC\Api\RequestBuilders\Xml;
+namespace FINDOLOGIC\Api\RequestBuilders\Xml20;
```

Comment:
Sounds like there may be new XML output versions in the future :smile:

Reply:
Who knows ;)